opentok/opentok-web-samples
# OPENTOK-51006 - 	 Create a Captions API sample app

This PR adds a sample app for captions API for the client side.

To test this app:
* Make sure you have a server side running - this can be found at https://github.com/opentok/learning-opentok-node.
* In Basic-Captions/js/config.js, change the URL to `http://localhost:8080`
* In Basic-Captions folder, start a local development server, such as `python3 -m http.server` or whatever you prefer.
* Navigate to `localhost:8000` from two tabs
* Open console logs
* Start the captions by clicking a button and start talking - notice the captions flowing.
* Stop captions from either one of the tabs.
* Notice no errors related to captions (you might see an error but that is related to missing `favicon.ico`) 

Note: the server side sample will be added separately to this repo - https://github.com/opentok/learning-opentok-node.
